mobile/src/libs/components/sign-background/styles.ts

```diff
@@ -0,0 +1,34 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  wrapper: {
+    backgroundColor: '#507CEB',
```

Comment:
Change backgroundColor to value from AppColor enum

Reply:
For background it's ok) we will use it only once

Reply:
yeah - but since the color is already in the enum under BLUE_200, I guess we can use it, right? 

```
const AppColor = {
  BLUE_100: '#4E80FF',
  BLUE_200: '#507CEB',
  PINK_100: '#E1A7D5',
  RED_100: '#D51A52',
  GRAY_100: '#EBF3FF',
  GRAY_200: '#CCDDF3',
  GRAY_300: '#ACB4D0',
  GRAY_400: '#302E36',
  GRAY_500: '#1D1C21FF',
  TRANSPARENT_GRAY_100: '#FFFFFF7F',
  TRANSPARENT_GRAY_200: '#312F377F',
} as const;```

Reply:
then yes, you are right)